content/updates/index.json

```
@@ -6,7 +6,7 @@
   },
   "2020-10": {
     "title": "🎃 October Serenity updates",
-    "description": "New Serenity updates of October",
+    "description": "This fall, quickly automate your content creation with the rules engine, easily work with product variants and models thanks to our product grid enhancements, better prioritize your data quality focus areas and benefit from other improvements.",
```

Still too short @LaureBro. I suggest adding:
```suggestion
    "description": "This fall, in October, quickly automate your content creation with the rules engine, easily work with product variants and models thanks to our product grid enhancements, better prioritize your data quality focus areas and benefit from other improvements.",
```

---

```suggestion
    "description": "The month of October rhymes with better productivity, quickly automate your content creation with the rules engine, easily work with product variants and models thanks to our product grid enhancements, better prioritize your data quality focus areas and benefit from other improvements.",
```

---

```suggestion
    "description": "The month of October rhymes with better productivity, quickly automate your content creation with the rules engine, easily work with product variants and models thanks to our product grid, better prioritize your data quality focus areas and benefit from other improvements.,
```

---

```suggestion
    "description": "October rhymes with better productivity, quickly automate your content creation with the rules engine, easily work with product variants and models thanks to our product grid enhancements, better prioritize your data quality focus areas and benefit from other improvements.",
```